### PR TITLE
Slightly condenses the AI tracking href

### PR DESCRIPTION
**Pull request by DominikPanic**
It's not /tg/ but it sure is a hella lot cleaner

### CODE REVIEW

File: code/game/say.dm [hunk 1]
```diff
@@ -50,8 +50,9 @@ GLOBAL_LIST_INIT(freqtospan, list(
 	//Start name span.
 	var/spanpart2 = "<span class='name'>"
 	//Radio freq/name display
-	var/job = speaker.GetJob()
-	var/freqpart = radio_freq ? "\[[get_radio_name(radio_freq)][job ? " ([job])": ""]\] " : ""
+	var/freqpart = compose_freq(speaker, radio_freq)
+	//Job display
+	var/jobpart = compose_job(speaker, message_language, raw_message, radio_freq)
```
Comment (Rohesie) [inline code/game/say.dm]: Why not defines?

Comment (DominikPanic) [reply code/game/say.dm]: Give an example and I'll implement it.

Comment (Rohesie) [reply code/game/say.dm]:
```
#define CMSG_FREQPART compose_freq(speaker, radio_freq)
#define CMSG_JOBPART compose_job(speaker, message_language, raw_message, radio_freq)
/atom/movable/proc/compose_message(atom/movable/speaker, datum/language/message_language, raw_message, radio_freq, list/spans, message_mode, face_name = FALSE)
(...)
return "[spanpart1][spanpart2][CMSG_FREQPART][languageicon][CMSG_JOBPART][namepart][endspanpart][messagepart]"
(...)
#undef CMSG_FREQPART
#undef CMSG_JOBPART
```

Comment (DominikPanic) [reply code/game/say.dm]: I don't really see it as being cleaner, but if you really want it, I can do it

**Pull request merged**